app/services/vot/parser.rb

```diff
@@ -113,16 +110,6 @@ def validate_component_uniqueness!(component_values)
       end
     end
 
-    def raise_unsupported_component_exception(component_value_name)
```

Comment:
I'll remark here that this may be a deviation from the spec since we are accepting vectors we do not recognize that are not described by our trustmark. However, that may not be a huge deal since this API should be going away soon.

Reply:
thanks for that note! i could probably update it to only raise when vectors of trust are passed in, if you have any feelings about it! (although i think we are deviating from the spec in some ways anyways, like having implied vectors, and as you said, it's hopefully not long for our code base.)